samples/subsys/usb/audio/headphones_microphone/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _usb_audio_headphones_microphone:
+
+USB Audio Sample Application
+###################################
+
+Overview
+********
+
+This sample app demonstrates use of a USB Audio driver by the Zephyr
+project. This very simple sample that performs loopback over IN/OUT
+ISO endpoints. The device will show up as two audio devices. One
+Input (Microphone) and one Output (Headphones) device.
+
+Building and Running
+********************
+
+In order to build the sample an overlay file with required options
+must be provided. By default app.overlay is added. An overlay contains
+software and hardware specific information which allow to fully
+describe the device.
+
+After you have built and flashed the sample app image to your board, plug the
+board into a host device, for example, a PC running Windows.
```

Comment:
Window OS or better does not claim on a specific OS as it should work on Linux Host, Window OS, Mac OS....